api/v1alpha1/selfnoderemediationconfig_types.go

```diff
@@ -114,6 +114,12 @@ type SelfNodeRemediationConfigSpec struct {
 	// It will be ignored when empty (which is the default).
 	EndpointHealthCheckUrl string `json:"endpointHealthCheckUrl,omitempty"`
 
+	// HostPort is used for internal communication between SNR agents.
+	// +optional
+	// +kubebuilder:default:=30001
+	// +kubebuilder:validation:Minimum=1
```

Comment:
Should we also enforce only 1 port value here?

Reply:
I don't follow :sweat_smile: 
ATM 30001 is hard coded, the suggest change make it configurable while leaving 30001 as the default choice.
What do you suggest ?

Reply:
The configuration should also allow no more than 1 value? 
I saw in the code you also check that the list contains only 1 value (in a test), so maybe we can also set `maximum=1`?

Reply:
I think I understand now, IIUC there is a bit of a mix-up.
It is true that container may have several ports, but it's not relevant to this PR because we create this container according to the `ds` and it will always have one port.
This validation is about the configuration value  of that port (i.e HostPort) and not the number of ports  - so in a nutshell it states that the configured port must have a positive number.

Reply:
Oh, I misunderstood then, thanks for the clarification!